kong/cli/services/serf.lua

```diff
@@ -28,7 +28,11 @@ function Serf:_get_cmd()
   local cmd, err = Serf.super._get_cmd(self, {}, function(path)
     local res, code = IO.os_execute(path.." version")
     if code == 0 then
-      return res:match("^Serf v0.7.0")
+      local version_match = res:match("^Serf v0.7.0")
+      if not version_match then
+        logger:warn("Incompatible Serf version in the path. Required version is Serf v0.7.0")
+      end
+      return version_match
```

Comment:
I was leaving room for the possibility that you have `serf` multiple times in your path. In this case it might be possible that the first version encountered is `0.6.4` but the second one is `0.7.0`. I'm not sure how the rest of the app would work with that sort of state, but it's possible it would cause a breaking change for a small section of poorly configured users :disappointed: